core/scss/components/cta/_cta.scss

```diff
@@ -25,17 +25,17 @@
 
   .su-cta__icon {
     display: block;
+    max-width: 30%;
     position: absolute;
     top: 50%;
     left: 50%;
     $_btn-height: $lead-font-size + 3rem; //height of the button
     $_shift: calc(#{-50%} - #{$_btn-height / 2}); //calculate the extra shift up for the icon to compensate for the button height
     transform: translate(-50%, $_shift);
-    color: #fff;
-    font-size: 4em;
-    text-shadow: 0 0 10px rgba(0, 0, 0, .4);
+    //color: #fff;
```

Comment:
If you don't need something please remove it. Don't leave unneeded code behind.

Reply:
Sure - removed.